src/pybreaker.py

```diff
@@ -106,21 +106,31 @@ def reset_timeout(self, timeout):
         """
         self._reset_timeout = timeout
 
+    def _create_new_state(self, state, notify=True):
+        """
+        Return state object from state string, i.e.,
+        'closed' -> <CircuitClosedState>
+        """
+        state_map = {
+            STATE_CLOSED: CircuitClosedState,
+            STATE_OPEN: CircuitOpenState,
+            STATE_HALF_OPEN: CircuitHalfOpenState,
+        }
+        try:
+            return state_map[state](self, state, notify=notify)
+        except KeyError:
+            msg = "Unknown state {!r}), valid states: {}"
+            raise ValueError(msg.format(state, ', '.join(state_map)))
```

Comment:
Just thinking whether it's best to raise an error here (and cause the `CircuitBreaker` creation to fail, probably crashing the app in the process) or "fixing" it by assuming a Closed state in this case (and also logging it so that this won't happen silently).

Reply:
In my opinion as an application developer, my preference would be to cause the app to fail. If something is providing an unknown value here, I would think something has gone horribly wrong and an exception is warranted.  I think it's ok for the library to be strict here and remove ambiguity from the code.

Reply:
Agreed with @jcwilson, explicit failures are always preferable to me.